docs/source/implementations.rst

```diff
@@ -116,6 +116,15 @@ netjsongraph.js
 library based on the popular d3.js visualization framework which can be used to
 visualize NetJSON ``NetworkGraph`` objects.
 
+MeshNetSimulator
+^^^^^^^^^^^^^^^

```

Comment:
just add one more `^` under the `r` otherwise python-sphinx may not render it correctly, thanks